src/js/TextFields/TextField.js

```diff
@@ -428,7 +430,11 @@ export default class TextField extends PureComponent {
     const multiline = this._isMultiline(nextProps);
     if (this._isMultiline(this.props) !== multiline) {
       this._updateMultilineHeight(nextProps);
-      window[`${multiline ? 'add' : 'remove'}EventListener`]('resize', this._updateMultilineHeight);
+      if (multiline) {
+        this.elementResizeDetector.listenTo(this._node, this._updateMultilineHeight);
```

Comment:
Where does the `elementResizeDetector` come from? Should it be the `_resizeObserver` instead?